src/Rootfs.jl

```diff
@@ -544,7 +544,7 @@ consists of four shards, but that may not always be the case.
 """
 function choose_shards(p::AbstractPlatform;
             compilers::Vector{Symbol} = [:c],
-            rootfs_build::VersionNumber=v"2022.3.5",
+            rootfs_build::VersionNumber=v"2022.4.15",
```

Comment:
We should be able to use the new https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/67ad9e77d2c31e51681b7b9b31c480b056bec4ab/src/Rootfs.jl#L387-L388 to automatically get the version number